README.md

```diff
@@ -11,3 +11,82 @@ repositories.
+* OpenSSL - software library and tool containing highly optimized version of basic cryptography primitives
+* journald - system service for collecting and storing log data, introduced with `systemd`
+* zlib - software library used for data compression
+
+#### Installing required tools on different Linux distributions
+Open a terminal window and execute distribution specific commands under regular user with elevated priviledges using `sudo`.
+
+**Debian** / **Ubuntu**
+
+```bash
+sudo apt-get update # Update metadata for backages provided by your Linux distribution
+sudo apt-get install -y curl git libssl-dev libsystemd-dev zlib1g-dev # Install required libraries and their header files required to build software
+```
+**RHEL** / **CentOS** / **Fedora**
+
+```bash
+sudo yum install -y curl git openssl-devel systemd-devel zlib-devel # Install required libraries and their header files required to build software
+```
+
+**SLES** / **openSUSE**
+
+```bash
+sudo zypper install -y curl gcc git gmp-devel libopenssl-devel systemd-devel zlib-devel # Install required libraries and their header files required to build software
+```
+
+#### Installing `stack` software building toolkit on all Linux distributions
+Execute `curl --proto '=https' --tlsv1.2 -sSf https://get.haskellstack.org/ | sh -s --` in the open terminal window and enter `root` password when prompted:
+
+```bash
+ladm_mark_stopka@mark-asus-zenbook-s:~> curl --proto '=https' --tlsv1.2 -sSf https://get.haskellstack.org/ | sh -s --
+Detected Linux distribution: opensuse
+
+This installer doesn't support your Linux distribution, trying generic
+bindist...
+
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100   628  100   628    0     0   1764      0 --:--:-- --:--:-- --:--:--  1759
+100 7404k  100 7404k    0     0  3150k      0  0:00:02  0:00:02 --:--:-- 5971k
+Installing Stack to: /usr/local/bin/stack...
+
+About to use 'sudo' to run the following command as root:
+    install -c -o 0 -g 0 -m 0755 /tmp/tmp.h54iH2BT2W/stack /usr/local/bin
+in order to copy 'stack' to the destination directory.
+
+[sudo] password for root:
+```
+
+#### Make a copy of `cardano-node` source-code repository with Git
+Use Git to make a `clone` of `cardano-node` GitHub repository and switch into the local copy, in open terminal window, run:
+```bash
+git clone https://github.com/input-output-hk/cardano-node/
+cd cardano-node
+```
+
+#### Initiate `cardano-node` build with `stack`
+Use [Haskell Stack](https://docs.haskellstack.org/en/stable/README/) to build required `cardano-node` binaries, there are 4 executable targets defined for `cardano-node`, however only `cardano-cli` and `cardano-node` are required to connect to the Cardano Blockchain network. In open terminal window, run:
+
+```bash
+stack build \
+      --flag cardano-node:-with_wallet-client \
```

Comment:
@dcoutts requested change of a flag name to `with_wallet_client` in #597 which has been done, can you please make the two-liner change (also line 87)?

Reply:
@mark-stopka Sorry, where does that change go? Ad diff posted here would be very useful.